README.md

```diff
@@ -22,10 +22,12 @@ import (
 
 func main() {
   r, err := rabbus.NewRabbus(rabbus.Config{
-    Dsn      : "amqp://guest:guest@localhost:5672",
-    Attempts : 5,
-    Sleep     : time.Second * 2,
-    Durable  : true,
+    Dsn         : "amqp://guest:guest@localhost:5672",
+    Durable     : true,
+    Retry       : rabbus.Retry {
+      Attempts  : 5,
+      Sleep     : time.Second * 2,
+    },
```

Comment:
Could you add the example of how to use `Breaker`?

Reply:
Sure 👍